src/App.jsx

```diff
@@ -12,6 +13,7 @@ function App() {
         <Route exact path="/" component={Home} />
         <Route path="/question/:question_number" render={props => <Question {...props} />} />
         <Route path="/outcome/:outcome_number" render={props => <OutcomeMessage {...props} />} />
+        <Raven dsn="https://174a79db7cf54ad2a03bafd2a075af8b@sentry.io/482571" />
```

Comment:
doesn't this need to be injected via the pipeline? or will all apps (local,  staging etc) track this in the same way?

Reply:
All of the stages share the same dsn, it then automagically detects environment.

Reply:
✨ Automagically ✨